pkg/kubelet/dockershim/security_context.go

```diff
@@ -113,6 +113,14 @@ func modifyHostConfig(sc *runtimeapi.LinuxContainerSecurityContext, hostConfig *
 	}
 	hostConfig.SecurityOpt = append(hostConfig.SecurityOpt, apparmorSecurityOpts...)
 
+	if sc.NoNewPrivs {
+		// "no-new-privileges" was added in Docker 1.11
+		// which is API version 1.23
+		if apiVersion.GTE(semver.Version{Major: 1, Minor: 23}) {
```

Comment:
So I am not sure exactly how we would do this other than checking the kernel version is >= 3.5